packages/raccoon-city-ui/src/components/Main/Contacts/Contact/ContactForm.tsx

```diff
@@ -61,16 +62,13 @@ export function ContactForm({onClose, contact}) {
     const {data, loading, error} = useQuery(GET_USERS);
     const {data: dropdowns, loading: dropdownsLoading} = useQuery(GET_CONTACT_DROPDOWNS);
 
-    const {data: trades} = useQuery(APPROPRIATE_TRADES, {
+    const {data: trades, loading: isLoadingTrade} = useQuery(APPROPRIATE_TRADES, {
         fetchPolicy: 'cache-and-network',
         variables: {
             contactId: contact.id
         }
     });
 
```

Comment:
I think it is better to query appropriate trades directly in ContactTrades but not in a parent component :)

Reply:
make sense